.pylintrc

```diff
@@ -51,7 +51,7 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=W0613,I0021,I0020,C0111,R0902,R0903,W0231,W0611,R0913,W0703,I0011,R0904,R0205,R1705,R1710,C0415,R1725,W0707,R1732,W1512,C0209,W1514,C0412,C0411,R1735
+disable=W0613,I0021,I0020,C0111,R0902,R0903,W0231,W0611,R0913,W0703,I0011,R0904,R0205,R1705,R1710,C0415,R1725,W0707,R1732,W1512,C0209,W1514,C0412,C0411,R1735,R0917
```

Comment:
> [!NOTE]
> This PR adds `R0917` to the `disable` list to prevent the following pylint errors experienced after the version updated:
> ```
> chalice/package.py:1456:4: R0917: Too many positional arguments (6/5) (too-many-positional-arguments)
> ```
> I want to avoid changing the signatures for all flagged functions.